app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -266,7 +268,15 @@ class EventDetailsFragment : Fragment() {
             Picasso.get()
                     .load(it)
                     .placeholder(R.drawable.header)
-                    .into(rootView.eventImage)
+                    .into(rootView.eventImage, object : Callback {
```

Comment:
Why

Reply:
To know that the image is loaded successfully, so it'll not send a placeholder image when failed.

Reply:
Remove callback in onDestroy

Reply:
`Picasso.get().cancelRequest(rootView.eventImage)`
will this work?

Reply:
Should